next/CMakeLists.txt

```diff
@@ -0,0 +1,924 @@
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+project("Mapbox GL Native" LANGUAGES CXX C)
+
+set(MBGL_ROOT ${PROJECT_SOURCE_DIR}/..)
```

Comment:
I think `PROJECT_SOURCE_DIR` won't work in situations where GL native is a dependency of an external project(). I'd go with `CMAKE_CURRENT_SOURCE_DIR` instead.

Reply:
It works, I tested. :-) It the directory of the most recent `project()`

```
Top level source directory for the current project.

This is the source directory of the most recent project() command.
```